cfpb/cfgov-refresh
# P4C: IE11 fixes

Enable IE11 polyfills for college costs app.

## Additions

- Polyfill for `matches` functionality
- Use `corejs` to handle additional polyfills 

## How to test this PR
1. Open branch in IE11 by either
  a. Checking out branch, running `./frontend.sh`, and navigating to [college costs](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/) in IE11 in Saucelabs, or
  b. Navigate to `/paying-for-college/your-financial-path-to-graduation/` on DEV2 in IE11
2. Click `getting started` button and verify that app runs as expected


## Screenshots
### College costs running in IE11

<img width="1100" alt="Screen Shot 2020-07-23 at 6 55 55 AM" src="https://user-images.githubusercontent.com/778171/88294950-ac498b00-ccb1-11ea-82fd-ba2ccfa80fc5.png">


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
